cmd/tuf/import_key.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/flynn/go-docopt"
+	"github.com/theupdateframework/go-tuf"
+	"github.com/theupdateframework/go-tuf/data"
+	"github.com/theupdateframework/go-tuf/pkg/keys"
+)
+
+func init() {
+	register("import-key", cmdImportKey, `
+usage: tuf import-key [--expires=<days>] <role> <file>
+
+Import an existing signing key for the given role.
+
+The key will be imported from the file and added to the "keys" directory with
```

Comment:
The bit about the "keys" directory is an implementation detail IMO and we don't want to commit to that in the public docs. I think the important points to hit are:

- the key will be imported to this repo's list of keys
- the key will be added to the role's keys

Reply:
Improved the documentation to remove implementation details.